src/App/AppEffects.h

```diff
@@ -223,39 +224,45 @@ class Frizzles2D:public Effect {
   }
 }; // Frizzles2D
 
+#define NUM_GEQ_CHANNELS 16                                           // number of frequency channels. Don't change !!
+
 class GEQEffect:public Effect {
 public:
+  byte previousBarHeight[1024];
+  uint8_t intensity;
+  uint8_t speed;
+  bool check1;
+  bool check2;
```

Comment:
Should be named as per their function in the effect, not based on WLED param names

Reply:
This is quick and dirty, I was thinking of a function like getControls which calls initSlider(..., control name, ...)

Similar as now done in fx.chFun

This function does not claim room of variables in the class.

Only drawback is that the effect itself should call getValue("control name") which is too slow, so maybe store that value during setup(), but then again we create local class variables...